packages/containers/src/List/List.container.js

```diff
@@ -48,7 +48,7 @@ class List extends React.Component {
 			sort: PropTypes.object,
 			filter: PropTypes.object,
 			pagination: PropTypes.shape({
-				onChange: PropTypes.func,
+				onChange: PropTypes.string,
```

Comment:
answer is here https://github.com/Talend/ui/blob/master/packages/containers/src/List/List.container.js#L168